CHANGE_LOG

```diff
@@ -1,3 +1,34 @@
+
+Version 0.21.0
+--------------
+
+This release introduces support for AMD's Heterogeneous System Architecture,
+which allows running the same code both on the CPU and the GPU.
```

Comment:
The HSA programming model is basically identical to CUDA, and there is not a CPU target for executing an HSA function.  The primary distinction with HSA at this point is a CPU/GPU shared memory space.